commons/commons-bio/src/main/kotlin/ebi/ac/uk/io/sources/FilesSources.kt

```diff
@@ -4,10 +4,10 @@ import ebi.ac.uk.extended.model.ExtFile
 import ebi.ac.uk.model.Attribute
 import java.io.File
 
-class ComposedFileSource(private val sources: List<FilesSource>) : FilesSource {
```

Comment:
basically, because this was hidden the fact that actually, we have many sources. I notice this abstraction issue as when I was going to trigger an error only one source was able which does not represent correctly business case.